app/models/profile.rb

```diff
@@ -4,6 +4,9 @@ class Profile < ApplicationRecord
   validates :data, presence: true
   validates :user_id, uniqueness: true
   validates :location, :website_url, length: { maximum: 100 }
+  validates :website_url,
+    format: { with: URI::DEFAULT_PARSER.make_regexp(%w[https http]) },
+    if: Proc.new {|p| p.website_url.present? }
```

Comment:
We don't need this `if:` we can use `allow_nil: true` instead, the specs are still green after making this change locally.

```suggestion
  validates :website_url,
            format: { with: URI::DEFAULT_PARSER.make_regexp(%w[https http]) },
            allow_nil: true
```

Additional nitpick: with the current formatting Rubocop complains about the alignment of the arguments to the `validates` method call. This line and the next should be aligned with `:website_url`.

Reply:
Sorry @djuber, I must have commented on the commit instead of the PR.

My vote goes to using `validates :field, url: {}`

see https://github.com/forem/forem/blob/dc27f597fb7418ba126c3f05a19fa49f2f22bde3/app/models/article.rb#L62

Reply:
well that's even easier - thank you both!

Reply:
Thanks @rhymes, I always forget about the URL validation gem we're using 😅